proto/svc-storage-grpc-flight_plan.proto

```diff
@@ -0,0 +1,99 @@
+syntax = "proto3";
+package grpc.flight_plan;
+
+import "google/protobuf/empty.proto";
+import "google/protobuf/field_mask.proto";
+import "svc-storage-grpc.proto";
+
+//FlightPlanRpc service
+service FlightPlanRpc {
+    rpc flight_plans(grpc.SearchFilter) returns (FlightPlans);
+    rpc flight_plan_by_id(grpc.Id) returns (FlightPlan);
+    rpc insert_flight_plan(FlightPlanData) returns (FlightPlan);
+    rpc update_flight_plan(UpdateFlightPlan) returns (FlightPlan);
+    rpc delete_flight_plan(grpc.Id) returns (google.protobuf.Empty);
+}
+
+//Flight Status Enum
+enum FlightStatus {
+    //READY
+    READY = 0;
+    //BOARDING
+    BOARDING = 1;
+    //IN_FLIGHT
+    IN_FLIGHT = 3;
+    //FINISHED
+    FINISHED = 4;
+    //CANCELLED
+    CANCELLED = 5;
+    //DRAFT
+    DRAFT = 6;
+}
+//Flight Priority Enum
+enum FlightPriority {
+    //LOW
+    LOW = 0;
+    //HIGH
+    HIGH = 1;
+    //EMERGENCY
+    EMERGENCY = 2;
+}
+
+//FlightPlan
+message FlightPlan {
+    //id
+    string id = 1;
+    //data
+    FlightPlanData data = 2;
+}
+
+message UpdateFlightPlan {
+    //id
+    string id = 1;
+    FlightPlanData data = 2;
+    google.protobuf.FieldMask mask = 3;
+}
+
+//FlightPlanData
+message FlightPlanData {
+    //pilot_id
+    string pilot_id = 2;
+    //vehicle_id
+    string vehicle_id = 3;
+    //cargo
+    repeated uint32 cargo = 4;
```

Comment:
cargo what? assuming weight in grams from the uint32 type

Reply:
fixed in https://github.com/Arrow-air/svc-storage/pull/4/commits/e722fb189299be5e2547cac1e5f4310503067828